templates/default/sv-jenkins-run.erb

```diff
@@ -9,7 +9,7 @@
 exec 2>&1
 cd <%= node['jenkins']['master']['home'] %>
 touch jenkins.start
-exec chpst -u <%= node['jenkins']['master']['user'] %> -U <%= node['jenkins']['master']['user'] %> \
+exec chpst -u <%= node['jenkins']['master']['user'] %>:jenkins:docker:rvm -U <%= node['jenkins']['master']['user'] %>:jenkins:docker:rvm \
```

Comment:
This change should not be part of this PR. I believe the issue will be fixed as part of https://github.com/chef-cookbooks/jenkins/pull/390.